.travis.yml

```diff
@@ -1,18 +1,26 @@
-language: python
+language: c
 
-python:
-    - 2.7
+os:
+    - osx
+    - linux
 
-env: DISPLAY=:99 MOZ_XVFB=1
+compiler:
+    - gcc
+
+env:
+    - DISPLAY=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo ':99'; fi)
+      MOZ_XVFB=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo 1; fi)
+      BINARY=$(if [[ $TRAVIS_OS_NAME = 'osx' ]]; then echo '/Applications/FirefoxNightly.app/Contents/MacOS/firefox'; else echo 'firefox/firefox'; fi)
```

Comment:
Environment variables that you want to set on a per-OS basis and use in .travis.yml must go in its env: section for now. That's true for BINARY

I lean toward putting things  in .travis.yml whenever I can since it's the master config file, and this reduces the need to reference other files.